index.bs

```diff
@@ -10568,6 +10552,28 @@ class MyProcessor extends AudioWorkletProcessor {
 }
 </xmp>
 
+<h6 id="audioworkletprocess-callback-parameters" callback lt="AudioWorkletProcessCallback()">
+Callback {{AudioWorkletProcessCallback}} Parameters
+</h6>
+The {{AudioWorkletProcessCallback}} function has the following parameters:
+
+*Insert text here about how these FrozenArrays behave if they get transferred*.
```

Comment:
It will get reallocated when the array topology changes or it is transferred.

Reply:
Added some new text here.  PTAL.